### PR TITLE
Build for jdk 8 target.

### PR DESCRIPTION
**Pull request by aquamatthias**
Docker: rely on the official jdk 8 container and install mesos afterwards.
IntegrationTests: set max heap mem explicitly for all launched applications.

### CODE REVIEW

File: src/test/scala/mesosphere/marathon/integration/setup/ForwarderService.scala [hunk 1]
```diff
@@ -75,7 +75,7 @@ object ForwarderService {
     val conf = createConf(args: _*)
 
     ProcessKeeper.startJavaProcess(
-      s"app_${conf.httpPort()}",
+      s"app_${conf.httpPort()}", 128,
```
Comment (kolloch) [inline src/test/scala/mesosphere/marathon/integration/setup/ForwarderService.scala]: Please use a named parameter for the "128".

File: src/test/scala/mesosphere/marathon/integration/setup/ProcessKeeper.scala [hunk 2]
```diff
@@ -47,7 +47,7 @@ object ProcessKeeper {
     val workDirFile = new File(workDir)
     FileUtils.deleteDirectory(workDirFile)
     FileUtils.forceMkdir(workDirFile)
-    startJavaProcess("zookeeper", args, new File("."), sys.env, _.contains("binding to port"))
+    startJavaProcess("zookeeper", 256, args, new File("."), sys.env, _.contains("binding to port"))
```
Comment (kolloch) [inline src/test/scala/mesosphere/marathon/integration/setup/ProcessKeeper.scala]: named parameter, please

File: src/test/scala/mesosphere/marathon/integration/setup/ProcessKeeper.scala [hunk 3]
```diff
@@ -73,17 +73,18 @@ object ProcessKeeper {
     FileUtils.forceMkdir(mesosWorkDirFile)
 
     startJavaProcess(
-      "marathon", argsWithMain, cwd,
+      "marathon", 512, argsWithMain, cwd,
```
Comment (kolloch) [inline src/test/scala/mesosphere/marathon/integration/setup/ProcessKeeper.scala]: named parameter, please

**Comment by kolloch** [comment]
Besides my wish for named parameters, that looks excellent!

**Comment by kolloch** [comment]
My only concern is that our tests might not be reproducible because we just use the latest available mesos version. Maybe we can pin the version there?

**Comment by kolloch** [comment]
Maybe we address pinning mesos versions when we have proper cross-testing, though.

**Comment by aquamatthias** [comment]
Pinned the mesos version.
Added named parameters.

**Comment by kolloch** [comment]
LGTM!

**Pull request merged**